src/compiler/utilities.ts

```diff
@@ -8433,4 +8446,88 @@ namespace ts {
             return got;
         }
     }
+
+    // TODO: remove once tslint allows tsconfig.json to include lib "esnext.bigint"
+    /* @internal */
+    declare const BigInt: ((value: string) => number) | undefined;
+
+    /**
+     * Converts a bigint literal string, e.g. `0x1234n`,
+     * to its decimal string representation, e.g. `4660`.
+     */
+    export function parsePseudoBigInt(stringValue: string): string {
+        // Use native BigInt if available
+        if (typeof BigInt !== "undefined") {

```

Comment:
Can you please get rid of this. We would like to be unifying and always depend on our own string that will give consistent results in all cases.